crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -20,16 +21,6 @@ use std::ops::Mul;
 /// [`GlobalTransform`] is updated from [`Transform`] in the system
 /// [`transform_propagate_system`](crate::transform_propagate_system::transform_propagate_system).
 ///
-/// In pseudo code:
```

Comment:
Out of curiosity, why is this documentation block being removed?

Reply:
This came from this https://github.com/bevyengine/bevy/pull/2331#discussion_r651114152 part of the previous PR.